tdfc2/src/common/construct/initialise.c

```diff
@@ -1419,28 +1412,27 @@ get_aggr_elem(LIST(EXP) p, unsigned *ptag)
 */
 
 static EXP
-init_aggr_aux(TYPE t, CV_SPEC cv, LIST(EXP) *r, int start, IDENTIFIER id,
+init_aggr_aux(FieldIterator_t *sf_iter, TYPE t, CV_SPEC cv, LIST(EXP) *r, int start, IDENTIFIER id,
 	      ERROR *err)
 {
 	EXP e;
 	LIST(EXP) p = *r;
 	ERROR cerr = NULL_err;
 	CLASS_INFO ci = cinfo_none;
-	unsigned tag = TAG_type(t);
+	int tag = TAG_type(t);
```

Comment:
It seems like all other uses of TAG_type are unsigned — was there a reason for changing this to int? It doesn’t seem like any code using tag changed, but it’s also pretty late here so grain of salt.

Reply:
Whoops certainly a mistake.  I'll fix that.